.github/workflows/containerized_boefjes.yml

```diff
@@ -0,0 +1,69 @@
+name: Build containerized boefjes
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-*"
+    tags:
+      - "*"
+    paths:
+      - boefjes/boefjes/plugins/**
+      - boefjes/images/**
+      - .github/workflows/containerized_boefjes.yml
+  pull_request:
+    paths:
+      - boefjes/boefjes/plugins/kat_nmap_tcp/**
+      - boefjes/boefjes/plugins/kat_nmap_udp/**
+      - boefjes/boefjes/plugins/kat_dnssec/**
```

Comment:
That wouldn't work I believe since we use the (python and other) files in the whole directory, where I think your suggestion would only target changes in the Dockerfiles

Reply:
You're right!